MapboxNavigation/Waypoint.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import MapboxDirections
+
+extension Waypoint {
+    var location: CLLocation {
+        return CLLocation.init(latitude: coordinate.latitude, longitude: coordinate.longitude)
```

Comment:
Is this used anywhere?

Reply:
Used [here.](https://github.com/mapbox/mapbox-navigation-ios/pull/848/files#diff-292af689ce88d5b5aa06966456abc0e9R597)

Reply:
Nit: .init is implied here so it can be dropped.